### PR TITLE
Dynamic cards

### PR DESCRIPTION
**Pull request by lsimmons1832**

### CODE REVIEW

**Comment by lsimmons1832** [comment]
@devoisaurus

File: cards.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 Createbutton.addEventListener("click", createCard);
 
 function createCard() {
 	var textElement = document.getElementById("userInput");
-	//console.log("verify textElement", textElement.value);
-	// var textCard = textElement.value;
+
 	 var textCard = document.getElementById("output");
-	 textCard.innerHTML = "<div>" + textElement.value + "</div>";
+	 textCard.innerHTML = "<div>" + textElement.value + "</div><button id='delete'>Delete</button>";
```
Comment (devoisaurus) [inline cards.js]: It looks like you have the delete button on the text box rather than the individual cards.

**Comment by devoisaurus** [comment]
Ok, so it looks like when you add a new card, it overwrites the original card instead of adding a new one

**Comment by lsimmons1832** [comment]
@devoisaurus 
This isn't styled perfectly, but it meets the assignment requirements so I ran with it.  I'll revisit styling after I'm caught up on other assignments.

**Comment by lsimmons1832** [comment]
@devoisaurus

**Pull request merged**